gprofiler/utils.py

```diff
@@ -486,34 +486,29 @@ def random_prefix() -> str:
     return ''.join(random.choice(string.ascii_letters) for _ in range(16))
 
 
+CGROUPFS_ROOT = "/sys/fs/cgroup"  # TODO extract from /proc/mounts, this may change
+
+
 class CpuUsageLogger:
     NSEC_PER_SEC = 1000000000
 
-    def __init__(self, logger: logging.LoggerAdapter, cgroup: str, enabled: bool):
+    def __init__(self, logger: logging.LoggerAdapter, cgroup: str):
         self._logger = logger
-        self._cpuacct_usage = Path(f"/sys/fs/cgroup/{cgroup}cpuacct/cpuacct.usage")
-        self._enabled = enabled
+        self._cpuacct_usage = Path(f"{CGROUPFS_ROOT}/{cgroup}cpuacct/cpuacct.usage")
```

Comment:
The `//` doesn't harm. I prefer the `cgroup` to be `/`, it makes it very clear to be "root". Also, if we ever pass other cgroups, it's nicer to have `/path/to/cgroup` instead of `path/to/cgroup`.

I'll remove the extra `/`